### ISSUE TITLE
Bug: enchantments of %itemtype% expression not working

### ISSUE THREAD

**Issue by KingAlterIV** [SkriptLang/Skript#1693]
### Description

Broadcasting <none> when the item itself has a level after enchanting it.

### Steps to Reproduce

```
command /test:
	trigger:
		set player's held item to diamond sword
		set {_e} to player's held item
		enchant player's held item with sharpness 1
		broadcast "Held item - %enchantments of player's held item%"
		broadcast "Variable - %enchantments of {_e}%"
```
### Expected Behavior

I thought it was supposed to broadcast "Sharpness 1" but it didn't.

### Errors / Screenshots

https://i.imgur.com/mqUbrKM.png

https://gist.github.com/KingAlterIV/c2e42b57594a6dfa5ab0e9166bbecb43
### Server Information
* **Server version/platform:** 1.13.2       
* **Skript version:** Skript 2.3-beta2

**Comment by KingAlterIV** [comment]
Closed, duplicate of #1673

**Comment by KingAlterIV** [comment]
Actually, kind of different from #1673
